core/dropdowndiv.js

```diff
@@ -387,18 +372,17 @@ const showPositionedByRect = function(
  */
 const show = function(
     owner, rtl, primaryX, primaryY, secondaryX, secondaryY, opt_onHide) {
-  owner_ = owner;
-  onHide_ = opt_onHide || null;
+  owner = owner;
```

Comment:
Maybe make this newOwner to avoid shadowing?

Reply:
Whoops! Thought I caught all of these. I'll do another check.

Reply:
Ok I think this was actually the last one!